### DISCUSSION TITLE
Disable linting from the qwik optimizer

### DISCUSSION THREAD

**Discussion by wmertens** [QwikDev/qwik-evolution#188] [Proposals For Qwik]
### What is it about?

linting

#### Problems you are trying to solve:

Right now, by default, we lint files in dev mode. This slows down the build, especially on large projects.

---

#### What do you propose?

Remove linting from the plugin:

- IDEs have linting during editing
- if something breaks, it's obvious
- the build has a separate lint step anyway
- we only support eslint, not biome or whatever
- if someone wants this, they can install a plugin like https://github.com/nabla/vite-plugin-eslint

---

### Links / References

QwikDev/qwik#6837

**Comment by maiieul** [comment]
related to https://github.com/QwikDev/qwik/issues/6837. If we remove it, we don't need to add an explanation in the docs ;). Imo that plugin has done more harm than good. It also prints warning messages in the terminal even if the rule is disabled in an eslintrc.

**Comment by shairez** [comment]
Thanks @wmertens !

You know this already, but for the record - we decided to promote this into an RFC and as a first step - disable the linting (and if we'll see nobody is needing it, we can remove it in the future)
